### DISCUSSION TITLE
What plugin did you use for the winbar in the screenrecorder in the readme?

### DISCUSSION THREAD

**Discussion by vdegenne** [saghen/blink.cmp#450] [Q&A] [answered]
Sorry not directly a question related to blink but I'd like to know the plugin you(screenrecorder owner)'ve used in the video in the readme to show the open file in the top right corner.

Thanks!

**Comment by stefanboca** [comment, marked as answer]
I believe it's https://github.com/b0o/incline.nvim

**Reply by saghen** [reply]
Yeah, here's my config for it: https://github.com/Saghen/tuque/blob/6e03eeb0900000ed3bb07f25213f69c35ecfe584/lua/core/ui.lua#L60

**Reply by vdegenne** [reply]
Thanks
